requirements.txt

```diff
@@ -1,11 +1,11 @@
-Flask>=0.10.1
+Flask>=0.10.1,<1.1
 Flask-Cors>=3.0.2
 Flask-Compress>=1.4.0
 raven[flask]>=6.2.1
-pymongo>=3.0
-psycopg2
-requests
-python-dateutil
-pytz
-PyJWT
-bcrypt
+pymongo>=3.6,<3.7
+psycopg2>=2.7,<2.8
+requests>=2.18,<2.19
+python-dateutil>=2.7,<2.8
+pytz==2018.4
+PyJWT>=1.6,<1.7
+bcrypt>=3.1,<3.2
```

Comment:
What is the rational behind choosing these particular version ranges?

Reply:
I decided that it's better to use the latest versions, because they are often more stable and with fewer bugs. Also I'm sure that only changes in the `patch` version can guarantee backward compatibility. `major` and `minor` changes can just break something. 
What do you think?